Ash 42 improve accessibility and added CampgroundTypes

aeSzelag opened the pull request:
Pagination Visuals and CampgroundType 
Additions Improved pagination text on main page by bolding and changing font color to higher contrast. 
Noticed missing campgroundTypes, so I added those to the filtering menu on the main page. Added missing campgroundTypes to the viewCamp page. Issue #42
Add margin between text entry and search button margin added. Issue #42
Correct Filtering Menu Bug 
- cleared extra fetch call when refreshing the page 
- added selectedStates to useEffect Issue #42
Group labels in menu 
edited campList.js to connect the label in the filtering menus so that the word and the checkbox can both be clicked on for filtering. Issue #42
Menu width adjusted 
edited app.css to adjust the menu width to remove overlapping text. pr #42

aeSzelag committed:
Menu width adjusted
edited app.css to adjust the menu width to remove
overlapping text.
pr #42

aeSzelag committed:
Group labels in menu
edited campList.js to connect the label in the
filtering menus so that the word and the checkbox
can both be clicked on for filtering.
Issue #42

aeSzelag committed:
Correct Filtering Menu Bug
- cleared extra fetch call when refreshing the page
- added selectedStates to useEffect
Issue #42

aeSzelag committed:
Add margin between text entry and search button
margin added.
Issue #42

aeSzelag committed:
Pagination Visuals and CampgroundType Additions
Improved pagination text on main page by bolding
and changing font color to higher contrast.
Noticed missing campgroundTypes, so I added those
to the filtering menu on the main page.
Added missing campgroundTypes to the viewCamp page.
Issue #42

EvanMeyerEmich reviewed (approved):
Looks good!

swill93 reviewed (approved):
Everything looks good!

aeSzelag committed:
Fixed Filtering Menu Overlap
moved dropdown menu component into seperate file
for improved readability
Added function to reference when clicks are made
outside of the menu to signal menu to close,
removing overlapping issue and cleaning up the
page for better viewing.
Issue #42

aeSzelag committed:
Merge branch 'main' into ash-42-improve-accessibility

aeSzelag committed:
added margin to the Clear FIlters button
added CSS styling to add margin-bottom to the
Clear filters button.
Issue #42
